jest.config.js

```diff
@@ -10,6 +10,7 @@ const baseConfig = {
     'packages/experimental',
     'packages/legacy/src/resource',
   ],
+  testEnvironment: 'jsdom',
```

Comment:
this means you won't be running tests "like it's node", but more "like it's a browser".

i've found it works well on other projects to run tests twice, once with each testEnvironment. not a blocker ofc.

Reply:
Jest config returns array. Second one is node like you said.

Reply:
ah oops, didn't look at the whole file. nice :-)